SConstruct

```diff
@@ -779,6 +788,8 @@ noshell = has_option( "noshell" )
 
 jsEngine = get_option( "js-engine")
 
+rocksdb = get_option( "rocksdb" ) == "on"
```

Comment:
In Jenkins the build line is:

`scons CC=${CC} CXX=${CXX} --release --ssl --opt=on -j4 --use-sasl-client --tokubackup --wiredtiger --audit --rocksdb --PerconaFT CPPPATH=${INSTALLDIR}/include LIBPATH=${INSTALLDIR}/lib ${PSM_TARGETS}`